v6-glm-py/constants.py

```diff
@@ -0,0 +1,25 @@
+# Constants for main function arguments
```

Comment:
Maybe be a bit more elaborate what these mean.

Reply:
Added explanation for every constant